src/core/layout/qgslayoutexporter.cpp

```diff
@@ -491,6 +491,7 @@ QgsLayoutExporter::ExportResult QgsLayoutExporter::exportToPdf( const QString &f
   LayoutContextSettingsRestorer contextRestorer( mLayout );
   ( void )contextRestorer;
   mLayout->renderContext().setDpi( settings.dpi );
+  mLayout->renderContext().setFlags( settings.flags );
```

Comment:
Look two lines below all your changes ;)

Reply:
Ok, so the backport has not been yet done ?

Reply:
I'm unsure - I don't think this was introduced to fix the selection issue, it was introduced to fix font rendering issues.

BTW - it IS possible to test pdf/svg outputs, test_qgslayoutexporter.py has many pdf/svg checks.

Reply:
I have found your fix https://github.com/qgis/QGIS/commit/2752f8348a5060dae3834ca40f2645e92ec4b18a it is in master and in release-3_6.

I will update this PR to add some unit tests and I will propose a backport for release-3_4.